utils/unittest/fixtures.go

```diff
@@ -27,11 +27,21 @@ func AddressFixture() flow.Address {
 	return flow.Testnet.Chain().ServiceAddress()
 }
 
+func RandomAddressFixture() flow.Address {
+	// we use a 32-bit index - since the linear address generator uses 45 bits,
+	// this won't error
+	addr, err := flow.Testnet.Chain().AddressAtIndex(uint64(rand.Uint32()))
+	if err != nil {
+		panic(err)
+	}
+	return addr
+}
+
 func TransactionSignatureFixture() flow.TransactionSignature {
 	return flow.TransactionSignature{
 		Address:     AddressFixture(),
 		SignerIndex: 0,
-		Signature:   []byte{1, 2, 3, 4},
+		Signature:   SeedFixture(32),
```

Comment:
if this is meant to match the size of a tx signature, then it should be 64. We could also use the constant `crypto.SignatureLenECDSAP256`

Reply:
Updated in https://github.com/onflow/flow-go/pull/67/commits/dd0d566ac153278af33c2c175eb0b0ea2be772b7.